### PR TITLE
Improve CLI installation instructions

### PR DESCRIPTION
**Pull request by pdcastro**
References:
* CLI installation instructions: https://www.flowdock.com/app/rulemotion/r-product/threads/u-0Zurwif8-D8R_ghYbfpAghgF6
* `balenaCLI` styling: https://www.flowdock.com/app/rulemotion/r-product/threads/xrtkAC3uWDMJ4-4GkJL_Hmn8Jvb

Depends-on: balena-io/balena-cli/pull/2039

### CODE REVIEW

File: pages/learn/getting-started.md [hunk 1]
```diff
@@ -74,13 +74,9 @@ Once you have finished your image configuration, click the *Download {{ $names.o
...
-The recommended way to deploy code is to install the [{{ $names.cli.lower }}][cli]. The easiest way to do this is to use the installer for your OS available on the [releases page][releases]. Choose the latest release of the installer for your OS, and follow the [installation instructions][install].
-
-<img alt="{{ $names.company.lower }} CLI installer" src="/img/common/cli/cli-release.png" width="100%">
-
-__Note:__ You may also install the {{ $names.company.lower }} CLI via npm on a system running NodeJS, as explained in [NPM Installation][npminstall].
-
-To use the [{{ $names.cli.lower }}][cli], you need to login to your {{ $names.company.lower }} account. Login via the terminal using the `{{ $names.company.lower }} login` command:
+The recommended way to deploy code is using {{ $names.cli.lower }}. Follow the [balenaCli
+installation instructions][cli-install]. After it is installed, login to your {{ $names.company.lower}}
+account with the `{{ $names.company.lower }} login` command on the terminal:
```
Comment (pdcastro) [inline pages/learn/getting-started.md:79]: This is the main change that motivated this PR, as [discussed in Flowdock](https://www.flowdock.com/app/rulemotion/r-product/threads/u-0Zurwif8-D8R_ghYbfpAghgF6).

Note that this PR depends on balena-io/balena-cli/pull/2039, which simplifies the installation instructions in the balenaCLI repo, focusing on a single installation option for each operating system (while having a link for "advanced installation options" for users who are interested). The new instructions also contain a direct link to the latest release / version on GitHub.

The screenshot of balenaCLI's GitHub releases page was clickable and seemed to be more confusing than helpful, hence removed. Also removed was the note about the option of installing the CLI via NPM, which is certainly not a good idea for the getting started guide (too complex and not really recommended for most users).

File: config/names.coffee [hunk 2]
```diff
@@ -14,8 +14,8 @@ DOMAIN_ENGINE = process.env.DOMAIN_ENGINE || 'balena.io/engine'
 FORUMS_DOMAIN = process.env.DOMAIN_FORUMS || 'https://forums.balena.io'
 BASE_IMAGES_LIB = process.env.BASE_IMAGES_LIB || 'balenalib'
 BASE_IMAGES_CORE = process.env.BASE_IMAGES_CORE || 'balena'
-CLI_LOWER = process.env.CLI_LOWER || 'balena CLI'
-CLI_UPPER = (CLI_LOWER.charAt(0).toUpperCase() + CLI_LOWER.slice(1)) || 'Balena CLI'
+CLI_LOWER = process.env.CLI_LOWER || 'balenaCLI'
+CLI_UPPER = (CLI_LOWER.charAt(0).toUpperCase() + CLI_LOWER.slice(1)) || 'BalenaCLI'
```
Comment (pdcastro) [inline config/names.coffee:18]: A second commit styles "balena CLI" as "balenaCLI" (without space), and drops `'the'` when referring to it. Flowdock: https://www.flowdock.com/app/rulemotion/r-product/threads/xrtkAC3uWDMJ4-4GkJL_Hmn8Jvb

Comment (pdcastro) [reply config/names.coffee]: I have now removed this commit to make this PR easier to review, focusing on the CLI installation instructions. I will create a separate PR for the renaming.

File: pages/learn/getting-started.md [hunk 3]
```diff
@@ -74,13 +74,9 @@ Once you have finished your image configuration, click the *Download {{ $names.o
...
-The recommended way to deploy code is to install the [{{ $names.cli.lower }}][cli]. The easiest way to do this is to use the installer for your OS available on the [releases page][releases]. Choose the latest release of the installer for your OS, and follow the [installation instructions][install].
-
-<img alt="{{ $names.company.lower }} CLI installer" src="/img/common/cli/cli-release.png" width="100%">
-
-__Note:__ You may also install the {{ $names.company.lower }} CLI via npm on a system running NodeJS, as explained in [NPM Installation][npminstall].
-
-To use the [{{ $names.cli.lower }}][cli], you need to login to your {{ $names.company.lower }} account. Login via the terminal using the `{{ $names.company.lower }} login` command:
+The recommended way to deploy code is to use the {{ $names.cli.lower }}. Follow the [installation
+instructions][cli-install]. After it is installed, login to your {{ $names.company.lower}} account
+with the `{{ $names.company.lower }} login` command on the terminal:
```
Comment (pdcastro) [inline pages/learn/getting-started.md:79]: Flowdock discussion: https://www.flowdock.com/app/rulemotion/r-product/threads/u-0Zurwif8-D8R_ghYbfpAghgF6

This PR builds on balena-io/balena-cli/pull/2039 (merged) that simplified the installation instructions in the balenaCLI repo, focusing on a single installation option for each operating system. The new instructions also contain a direct link to the latest release / version on GitHub.

The screenshot of balenaCLI's GitHub releases page was clickable and seemed to be more confusing than helpful, hence removed. Also removed was the note about the option of installing the CLI via NPM, which is certainly not a good idea for the getting started guide (too complex and not really recommended for most users).

**Review by codewithcheese — APPROVED** [review]
thoughtful simplifications :+1:

**Pull request merged**